test/TestDummies/TestDummies.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.1;netstandard2.0;net462</TargetFrameworks>
```

Comment:
can't think of a specific reason a NS2.1 variant would add value? Or is the aim to pull in the NS2.1 variant of the prod assembly? (And if that's the aim, should there be a net48 one added instead of removing this?)